apps/st2-workflows/index.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Extreme Networks, Inc.
+//
```

Comment:
@m4dcoder Here is an example with the Copyright headers update we need for every file that includes Extreme Networks (c): 

```suggestion
// Copyright 2020 The StackStorm Authors.
// Copyright 2019 Extreme Networks, Inc.
//
```

See https://github.com/StackStorm/st2/pull/5003 for the `stackstorm/st2` analogy.

Reply:
Done